src/components/VideoPlayer/BaseVideoPlayer.js

```diff
@@ -145,8 +146,24 @@ function BaseVideoPlayer({
     }, [currentVideoPlayerRef, handleFullscreenUpdate, handlePlaybackStatusUpdate]);
 
     useEffect(() => {
+        if (!isUploading) {
+            return;
+        }
+        // If we are uploading a new video, we want to immediately set the video player ref.
```

Comment:
I use `isUploading` condition here because in attachment carousel with multiple video player on small screen, the video player ref is set to the wrong video player.

Reply:
For example, if we have video A and B in carousel, and we open video B, it could be possible the video A video player is set as the ref.